src/data/bluetooth.ts

```diff
@@ -84,7 +91,24 @@ export const subscribeBluetoothAdvertisements = (
     "_bluetoothDeviceRows",
     () => Promise.resolve<BluetoothDeviceData[]>([]), // empty array as initial state
 
-    subscribeUpdates,
+    subscribeBluetoothAdvertisementsUpdates,
     conn,
     onChange
   );
+
+export const subscribeBluetoothConnectionAllocations = (
+  conn: Connection,
+  onChange: (bluetoothAllocationsData: BluetoothAllocationsData[]) => void,
```

Comment:
In ZHA and ZwaveJS we use `callbackFunction` but you have already used `onChange` for bluetooth and there is no official guideline so just FYI, I guess

Reply:
a5e14395dc3c0aada06c01399faffc8ff26a9519